app/Policies/Training/WaitingList/WaitingListRetentionChecksPolicy.php

```diff
@@ -10,17 +10,38 @@ class WaitingListRetentionChecksPolicy
 {
     use HandlesAuthorization;
 
-    public function viewAny(Account $user): bool {}
-
-    public function view(Account $user, WaitingListRetentionCheck $waitingListRetentionCheck): bool {}
-
-    public function create(Account $user): bool {}
-
-    public function update(Account $user, WaitingListRetentionCheck $waitingListRetentionCheck): bool {}
-
-    public function delete(Account $user, WaitingListRetentionCheck $waitingListRetentionCheck): bool {}
-
-    public function restore(Account $user, WaitingListRetentionCheck $waitingListRetentionCheck): bool {}
-
-    public function forceDelete(Account $user, WaitingListRetentionCheck $waitingListRetentionCheck): bool {}
+    public function viewAny(Account $user): bool
+    {
+        return $user->hasAnyPermission('waiting-lists.access');
```

Comment:
I think we should constrain this to accesss of the waiting list type i.e. ATC access can see retention checks on ATC waiting lists.